Testing/Tests/data_loading_test.py

```diff
@@ -42,8 +44,8 @@ def test_visit_config():
     Ensure that DataLoading.py correctly produces a VisIt session file.
     """
     os.system("python PythonTool/DataLoading.py %s %s -s -v" % (geom_file, mesh_file))
-    diff = main.diff_files("VisitDefaultOutput.session", session_file)
-    assert len(diff) == 4
+    diff = subprocess.Popen(['diff', session_file, 'VisitDefaultOutput.session'], stdout=subprocess.PIPE)
```

Comment:
Is there a reason not to use `filecmp` here?

Reply:
I chose not to use `filecmp` because these two files are sure to have differences due to source file paths, the username, the host that VisIt is run on, etc. and `filecmp` simply returns True or False. I needed a tool that can return the actual differences in the file and then I can make sure they are the ones I expect! Also, the presence of `conftest.py` seems to be necessary for CircleCI to be able to run `pytest`, even though it is empty.